vector/src/main/java/im/vector/app/features/settings/VectorSettingsPreferencesFragment.kt

```diff
@@ -102,7 +102,7 @@ class VectorSettingsPreferencesFragment :
         }
 
         findPreference<Preference>(VectorPreferences.SETTINGS_PREF_SPACE_CATEGORY)!!.let { pref ->
-            pref.isVisible = !vectorFeatures.isNewAppLayoutEnabled()
+            pref.isVisible = !vectorFeatures.isNewAppLayoutFeatureEnabled()
```

Comment:
same, we want to check the labs flag here.

Reply:
Same as above